src/main/query_result.cpp

```diff
@@ -31,7 +31,7 @@ shared_ptr<FlatTuple> QueryResult::getNext() {
     return iterator->getNextFlatTuple();
 }
 
-void QueryResult::writeToCSV(string fileName) {
+void QueryResult::writeToCSV(string fileName, char delimiter, char escapeCharacter, char newline) {
```

Comment:
I think we should do a validation here to prevent user to use some dangerous/meaningless newline,escape,delimiter characters. I will let @mewim and you decide this.